tests/schemas/api_v3/blockchainAppsSchema.schema.js

```diff
@@ -45,6 +50,7 @@ const blockchainAppSchema = {
 		.positive()
 		.max(getCurrentTimestamp())
 		.required(),
+	escrow: Joi.array().items(escrow).min(0).required(),
```

Comment:
Since we're enforcing at least an entry for the LSK tokens for each sidechain.
```suggestion
	escrow: Joi.array().items(escrow).min(1).required(),
```